crates/bevy_remote/src/lib.rs

```diff
@@ -153,7 +154,148 @@
 //! - `has`: A map associating each type name from `has` to a boolean value indicating whether or not the
 //!   entity has that component. If `has` was empty or omitted, this key will be omitted in the response.
 //!
+//! ### Example
+//! To use the query API and retrieve Transform data for all entities that have a Transform
+//! use this query:
+//!
+//! ```json
+//! {
+//!     "jsonrpc": "2.0",
+//!     "method": "bevy/query",
+//!     "id": 0,
+//!     "params": {
+//!         "data": {
+//!             "components": ["bevy_transform::components::transform::Transform"]
+//!             "option": [],
+//!             "has": []
+//!        },
+//!        "filter": {
+//!           "with": [],
+//!           "without": []
+//!         },
+//!         "strict": false
+//!     }
+//! }
+//! ```
+//!
+//!
+//! To query all entities and all of their Reflectable components (and retrieve their values), you can pass in "all" for the option field:
+//! ```json
+//! {
+//!     "jsonrpc": "2.0",
+//!     "method": "bevy/query",
+//!     "id": 0,
+//!     "params": {
+//!         "data": {
+//!             "components": []
+//!             "option": "All",
```

Comment:
```suggestion
//!             "option": "all",
```
to match the comment above :)